torch/testing/_internal/hop_db.py

```diff
@@ -196,7 +197,23 @@ def body_fn(iter_t, x):
         check_batched_gradgrad=False,
         check_batched_forward_grad=False,
         check_inplace_batched_forward_grad=False,
-        supports_autograd=False,
+        supports_autograd=True,
+        # "torch.compile with aot_autograd does not currently support double backward."
+        supports_gradgrad=False,
+        skips=(
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestEagerFusionOpInfo",
+                "test_aot_autograd_exhaustive",
+                active_if=IS_WINDOWS,
+            ),
+            DecorateInfo(
```

Comment:
Probably need to add another DecorateInfo for another windows test `PYTORCH_OPINFO_SAMPLE_INPUT_INDEX=0 python test\test_ops_gradients.py -k TestBwdGradientsCPU.test_fn_grad_cond_simple_cpu_float64` based on the CI signal. probably something like:
```
            DecorateInfo(
                unittest.expectedFailure,
                "TestBwdGradientsCPU",
                "test_fn_grad",
                active_if=IS_WINDOWS,
            ),
```
 Can you also add a `TODO (yidi): figure out window test failures`?

Reply:
I added the DecorateInfo as well as a TODO

Reply:
The DecorateInfo should be:
```
DecorateInfo(
    unittest.expectedFailure,
    "TestBwdGradients",
    "test_fn_grad",
    active_if=IS_WINDOWS,
)
```

Reply:
I updated the DecorteInfo